resources/quality/ultimaker_s5/um_s5_aa0.4_um-petg_0.06mm.inst.cfg

```
@@ -0,0 +1,27 @@
+[general]
+definition = ultimaker_s5
+name = Extra Fine
+version = 4
+
+[metadata]
+material = ultimaker_petg
+quality_type = high
+setting_version = 22
+type = quality
+variant = AA 0.4
+weight = 1
+
+[values]
```

Need to enable the acceleration since it is now disabled by default
```suggestion
[values]
_plugin__curaenginegradualflow__0_1_0__gradual_flow_enabled = True
```